src/containers/trends/TrendsContainer.js

```diff
@@ -199,6 +199,9 @@ export class TrendsContainer extends React.Component {
       mostRecent: mostRecent.toISOString(),
       xScale: scaleLinear().domain([0, 864e5]),
       yScale,
+      selectDay: (date) => {
+        this.props.onSelectDay(datetime.midDayForDate(date, timePrefs));
+      },
```

Comment:
It can be, we just have to execute a function that returns a function that we can pass down so the `this` reference gets bound at this class instance.